editor/create_dialog.cpp

```diff
@@ -395,6 +396,11 @@ void CreateDialog::_confirmed() {
 		return;
 	}
 
+	TreeItem *selected = search_options->get_selected();
+	if (!selected->get_meta("__instantiable")) {
```

Comment:
```suggestion
	if (!selected->get_meta("__instantiable", true)) {
```
Prevents errors if for any reason the meta is not set (there aren't any right now).

Reply:
Not `if (!selected->get_meta(SNAME("__instantiable"), true)) {`?

Reply:
It's not called often, so it's not important here.